front/app/containers/Admin/formBuilder/components/FormResults/CompletionBar.tsx

```diff
@@ -33,14 +44,9 @@ const CompletionBar = ({
         )}
       </Box>
     )}
-    <Box
-      height="16px"
-      width="100%"
-      borderRadius="3px"
-      border={`1px solid ${colors.divider};`}
-    >
+    <CompletionBox>
       <Box height="100%" width={`${completed}%`} bgColor={bgColor} />
-    </Box>
+    </CompletionBox>
```

Comment:
Just wondering why we needed a styled div here rather than the Box component?? :)

Reply:
I needed to add an `overflow: hidden` property, this felt cleaner. Could I add it to a `Box` easily enough?

Reply:
Yep - we have that on the Box as well! :) it's used like this:

` <Box overflow='hidden'>`

Reply:
Ok that's loads easier - will update :)

Reply:
Awesome, thanks!! I'll take care of that merge conflict with master and push the change :+1:

Reply:
Sorted